src/components/InventoryManager/DeleteButton.js

```diff
@@ -1,29 +1,31 @@
 import React, { useState } from 'react';
-import Dialog from '@material-ui/core/Dialog';
-import DialogActions from '@material-ui/core/DialogActions';
-import DialogContent from '@material-ui/core/DialogContent';
-import DialogTitle from '@material-ui/core/DialogTitle';
 import Button from '@material-ui/core/Button';
 import { makeStyles } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
-import { green } from '@material-ui/core/colors';
-import AddIcon from '@material-ui/icons/Add';
-import AddListing from './AddListing';
+import DeleteOutlineSharpIcon from '@material-ui/icons/DeleteOutlineSharp';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import PropTypes from 'prop-types';
 
 const useStyles = makeStyles({
   button: {
-    float: 'right',
-    backgroundColor: green[500],
+    float: 'left',
+    backgroundColor: '#FFFFFF',
     '&:hover': {
-      backgroundColor: green[700],
+      backgroundColor: '#FAFAFA',
     },
+    border: '1px solid gray',
   },
   text: {
     fontFamily: 'Work Sans',
+    fontColor: '#000000',
```

Comment:
On my screen, the delete button is currently two lines, can we add css to make sure it has enough space to be one line?
<img width="441" alt="Screen Shot 2021-03-26 at 12 07 16 PM" src="https://user-images.githubusercontent.com/59102166/112681359-d698ad00-8e2b-11eb-8de4-4666d743912e.png">